requirements.txt

```diff
@@ -1,2 +1 @@
-goodtables==1.5.1
 tableschema>=1.0.3,<1.20.4
```

Comment:
is tableschema still needed?

Reply:
It's used in multiple modules of this plugin.

Reply:
should be swapped to https://framework.frictionlessdata.io/docs/framework/actions.html#describe which will output the format that the 'checker' now uses. it is similar  to tableschema is has extra parts.